ketten.csv

```diff
@@ -124,6 +124,14 @@ DE,supermarket,tegut,Q1547993,open_adapted,,,https://www.tegut.com/newsticker-co
 DE,supermarket,Wasgau,Q2536857,open_adapted,,,,https://wasgau-ag.de/wasgau-frischemarkt/
 CH,supermarket,Coop,Q432564,open,,,,https://www.coop.ch/de/unternehmen/standorte-und-oeffnungszeiten.html
 CH,supermarket,Migros,Q680727,open,,,,https://filialen.migros.ch/de
+CH,supermarket,Denner,Q379911,open,,,,https://www.denner.ch/de/filialen/
+CH,supermarket, Volg,Q2530746,open,,,,https://www.volg.ch/standorte-oeffnungszeiten/
+CH,supermarket,Aldi Suisse,Q125054,open,,,,https://www.aldi-suisse.ch/filialen/de-CH/Start
+CH,supermarket,LIDL Schweiz,Q151954,open,,,,https://www.lidl.ch/unternehmen/filialen-oeffnungszeiten/filialfinder
```

Comment:
Wie @datendelphin schon richtig schrieb, wären hier `Aldi` bzw. `LIDL` noch ein Tick besser.

Orte werden auf zwei Arten zugeordnet:
1. Über den Namen (Spalte 3)
2. Über die Wikidata-ID (Spalte 4)

Da möglicherweise nicht alle Orte mit Wikidata-ID getaggt sind, ist es eine gute Strategie, in der Namen-Spalte den gängigsten Namen zu verwenden, idealerweise den, der am häufigsten in OpenStreetMap verwendet wird.

Reply:
Habs korrigiert.